shell/platform/common/client_wrapper/include/flutter/method_channel.h

```diff
@@ -13,9 +13,14 @@
 #include "method_call.h"
 #include "method_codec.h"
 #include "method_result.h"
+#include "standard_method_codec.h"
 
 namespace flutter {
 
+static constexpr char kControlChannelName[] = "dev.flutter/channel-buffers";
+static constexpr char kResizeMethod[] = "resize";
+static constexpr char kOverflowMethod[] = "overflow";
```

Comment:
Why are these public? The channel serializations should be internal implementation details.